rustql-linker/src/main.rs

```diff
@@ -7,24 +7,38 @@ extern crate datafrog;
 extern crate rustql_common;
 extern crate serde_json;
 
+#[macro_use]
+extern crate log;
+extern crate env_logger;
+
 use rustql_common::data;
 use rustql_common::tuples;
 use std::fs::File;
+use std::error::Error;
 
 const USE_JSON: bool = false;
-
 const TARGET_DIR_VARNAME: &str = "EXTRACTOR_TARGET_DIR";
+const RUSTQL_DIR_VARNAME: &str = "/.rustql/crates";
+const DATABASE_FILE_VARNAME: &str = "database.db";
 
 fn main() {
-    let database = create_database();
+    env_logger::init();
 
-    save_database(&database, "database.db");
+    let database = create_database();
+    save_database(&database, DATABASE_FILE_VARNAME);
 }
 
 fn save_database(database: &tuples::Database, name: &str) {
-    let file = File::create(name).expect("could not create database file");
+    let file = match File::create(name) {
+        Err(e) => panic!("Could not create database {}: {}", name, e.description()),
+        Ok(file) => file,
+    };
+
     //serde_json::to_writer_pretty(file, database).expect("could not serialize to json");
-    bincode::serialize_into(file, database).expect("could not serialize to json");
+    match bincode::serialize_into(file, database) {
+        Err(e) => panic!("Could not write to database {}: {}", name, e.description()),
```

Comment:
You could use [unwrap_or_else](https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap_or_else) instead: `bincode::serialize_into(file, database).unwrap_or_else(|| panic!(...));`.